### ISSUE TITLE
Results on HPatches between pre-trained model and this Github

### ISSUE THREAD

**Issue by miumiu94** [eric-yyjau/pytorch-superpoint#99]
Hi, thanks for your great work!

I have a question. You showed results on HPatches that compare the evaluation from different models in the table. The first row shows the evaluation from pre-trained model which is from [SuperPointPretrainedNetwork](https://github.com/magicleap/SuperPointPretrainedNetwork), and the third row is from this GitHub. 

I notice that repeatability(in Detector metric) from this GitHub is better than that from pre-trained model, but matching score(in Descriptor metric) is lower distinctly. The result above mean that model from this GitHub can detect feature keypoint better, but match worse because descriptors have a poor descriptive ability. I observe that you use different batch size, learning rate, lamda-loss(desc loss/detect loss) from pre-trained model in the training details. I tried to adjust same parameters mentioned above with pre-trained model, but match performed even worse. 

So, I wonder why the result mentioned above occurred, and how to make the training reach agreement with the pre-trained model. 

Looking forward to receiving your reply!
